### DISCUSSION TITLE
(In)consistent annotation syntax

### DISCUSSION THREAD

**Discussion by jaschdoc** [flix/flix#6449] [General]
We allow some annotations to begin with lowercase characters, while some are only allowed to begin with upper case letters. Should we be more principled about this? E.g., allow both lowercase and uppercase letters for all annotations, just lowercase letters, or only uppercase letters?
Thoughts?

**Comment by magnus-madsen** [comment]
We should only allow uppercase annotations (but this requires a lot of renaming).

**Reply by mlutze** [reply]
`s/@test/@Test`

Easy, no?
